src/rezplugins/shell/_utils/powershell_base.py

```diff
@@ -77,53 +78,32 @@ def get_syspaths(cls):
             cls.syspaths = config.standard_system_paths
             return cls.syspaths
 
-        # detect system paths using registry
-        def gen_expected_regex(parts):
-            whitespace = r"[\s]+"
-            return whitespace.join(parts)
-
         # TODO: Research if there is an easier way to pull system PATH from
         # registry in powershell
         paths = []
 
         cmd = [
-            "REG", "QUERY",
-            ("HKLM\\SYSTEM\\CurrentControlSet\\"
-             "Control\\Session Manager\\Environment"), "/v", "PATH"
+            "powershell",
+            '(Get-ItemProperty "HKLM:SYSTEM/CurrentControlSet/Control/Session Manager/Environment").Path',
         ]
 
-        expected = gen_expected_regex([
-            ("HKEY_LOCAL_MACHINE\\\\SYSTEM\\\\CurrentControlSet\\\\"
-             "Control\\\\Session Manager\\\\Environment"), "PATH",
-            "REG_(EXPAND_)?SZ", "(.*)"
-        ])
-
         p = Popen(cmd, stdout=PIPE, stderr=PIPE,
                   shell=True, text=True)
```

Comment:
is it necessary for this to be a shell command now, is it not just a single arg passed to powershell binary?

Reply:
This is a good point and a leftover from the REG call. A quick test seemed to work and tests still passed locally.